android/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 4.3.0
+version: 4.3.1
```

Comment:
this is strange. https://github.com/hansemannn/titanium-firebase-config/blob/main/android/manifest#L5 should be there already, so this should be 4.3.2

Reply:
@jordanbisato Pull from master please